metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/FluxTopologyComponent.java

```diff
@@ -133,7 +138,25 @@ public void start() throws UnableToStartException {
   @Override
   public void stop() {
     if (stormCluster != null) {
-      stormCluster.shutdown();
+      try {
```

Comment:
Note here, we moved to storm 1.0.3 in order to have a single API that would work between storm 1.0.1 in the HDP profile as well as the default profile.  Storm 1.0.3 writes out worker artifact logs and they need to be cleaned up.